data/_models/shared/Datasets.csv

```diff
@@ -1,12 +1,19 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Datasets,,,,,,,,,,,dcat:Dataset,Definition of a dataset,DataCatalogueFlat,
-Datasets,,collection,ref,1,TRUE,,Collections,,,,dcterms:isPartOf,collection that these variables are part of,DataCatalogueFlat,
-Datasets,,name,,1,TRUE,,,,,,dcterms:title,unique dataset name in the model,DataCatalogueFlat,
+Datasets,,,,,,,,,,,"dcat:Dataset,dct:Dataset",Definition of a dataset,DataCatalogueFlat,
+Datasets,,collection,ref,1,TRUE,,Collections,,,,"dcterms:isPartOf,dct:inSeries",collection of which the dataset is part,DataCatalogueFlat,
+Datasets,,name,,1,TRUE,,,,,,"dcterms:title,dct:title,dct:identifier",unique dataset name in the model,DataCatalogueFlat,
 Datasets,,label,,,,,,,,,,short human readable description,DataCatalogueFlat,
-Datasets,,dataset type,ontology_array,,,CatalogueOntologies,Dataset types,,,,,"defines the type of dataset, e.g. genomic",DataCatalogueFlat,
+Datasets,,creator,ref_array,,,,Organisations,,,,dct:creator,"The organisation(s) responsible for creating the dataset",DataCatalogueFlat,
+Datasets,,publisher,ref_array,,,,Organisations,,,,dct:publisher,"The organisation(s) responsible for making the dataset available",DataCatalogueFlat,
+Datasets,,dataset type,ontology_array,,,CatalogueOntologies,Dataset types,,,,dct:type,"defines the type of dataset, e.g. genomic",DataCatalogueFlat,
 Datasets,,unit of observation,ontology,,,CatalogueOntologies,Observation targets,,,,,defines what each record in this table describes,DataCatalogueFlat,
-Datasets,,keywords,ontology_array,,,CatalogueOntologies,Keywords,,,,dcat:theme,enables grouping of table list into topic and to display tables in a tree,DataCatalogueFlat,
-Datasets,,description,text,,,,,,,,dcterms:description,description of the role/function of this table,DataCatalogueFlat,
+Datasets,,keywords,ontology_array,,,CatalogueOntologies,Keywords,,,,"dcat:theme,dct:theme",enables grouping of table list into topic and to display tables in a tree,DataCatalogueFlat,
+Datasets,,description,text,,,,,,,,"dcterms:description,dct:descriptions",description of the role/function of this table,DataCatalogueFlat,
+Datasets,,contact email,email,,,,,,,,dct:contactPoint,Contact e-mail address for this dataset,DataCatalogueFlat,
```

Comment:
dit zou ik niet doen. Is dit verplicht?

Reply:
`dct:contactPoint` is verplicht voor zowel _Datasets_ als _Dataset Series_

Reply:
can we 'extract/merge' this from the Contacts table? e.g. via Contacts.resource?

Reply:
This is what I discussed today, a way to associated info that is further away. 
Please remove 'email' from dataset and because I really don't want to have it here and create a new story to address this properly